runtime-decompiler/src/main/java/org/jrd/frontend/frame/main/MainFrameView.java

```diff
@@ -358,7 +375,7 @@ public void mouseClicked(MouseEvent mouseEvent) {
         // menuBar end
 
 
-        mainFrame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+        mainFrame.setDefaultCloseOperation(WindowConstants.DO_NOTHING_ON_CLOSE);
```

Comment:
@judovana I'm sorry, what? Why?

Reply:
That is to allow decision in the moment of closing. See  https://github.com/pmikova/java-runtime-decompiler/pull/165/files/d7217dc4ebf4c5612ba313d9e306bf8840b3e30e..705dd3531a8f4b18cb9c4d61cedfd0137d167f17#diff-f10a3125cb9a730838a5f954735836603897ee9e217935b5df83ea69090b89f0R291